src/assets/scripts/client/aircraft/AircraftCommander.js

```diff
@@ -367,14 +367,14 @@ export default class AircraftCommander {
      * @for AircraftCommander
      * @method runSID
      * @param aircraft {AircraftModel}
-     * @param data {array}
+     * @param data {array<string>} a string representation of the SID (may also include exit, eg sid.exit)
      * @return {array}   [success of operation, readback]
      */
     runSID(aircraft, data) {
-        const sidId = data[0];
+        const routeString = data[0].toUpperCase();
```

Comment:
Looks like this is because they are imported without amending the casing from the airport files. They are written in the files in upper case, and so when trying to look one up, they are stored in upper case and obviously won't match if we don't do this. I think SIDs/STARs/fixes/airways are the only cases where they are stored in upper case, all for the same reason. Since they are _always_ shown in upper case, it kinda makes sense to leave it upper case rather than casing it down just to case it back up later.

Reply:
If nothing else though, it seems safer to do the upper-casing in the FMS, where the comparison happens, that way other methods calling through different stacks will still get the right result regardless of their casing.

Reply:
Moved `.toUpperCase()` to the FMS.

Reply:
FYI if you type in something like `sid EDDF18.OBOKA1L.OBOKA` the route string will end up as "eddf18.oboka1l.oboka", i.e. lower case even though typed in as upper-case. And as a result of that, trying to find the corresponding procedure will always fail, because procedures are stored in upper-case. 

The upper-case/lower-case issue is the original bug reported in  #955. Anything else in this PR are changes that were requested later by @erikquinn.

Reply:
It works for now.

Created #1173 for @n8rzz's sanity 😄